test/typescript/asset/main.ts

```diff
@@ -27,17 +27,35 @@ function assets(scope: Construct, assetOverrides = {}) {
     type: AssetType.ARCHIVE,
   });
 
+  const relativeAsset = new TerraformAsset(scope, "relative-asset", {
+    path: "relative-asset.txt",
+    ...assetOverrides,
+  });
+
+  const relativeAssets = new TerraformAsset(scope, "relative", {
+    path: "./relative",
+    ...assetOverrides,
+  });
```

Comment:
I think this works only by accident, since the files are all present in `process.cwd()`. When referencing from a nested folder structure, this wouldn't work without inferring the file location.  When looking at other languages than Typescript, I'm not sure how this would work.